frontend/components/product-list/MetaTags.tsx

```diff
@@ -44,10 +45,13 @@ export function MetaTags({ productList }: MetaTagsProps) {
       productList.path
    }${itemTypeHandle}${page > 1 ? `?${PRODUCT_LIST_PAGE_PARAM}=${page}` : ''}`;
    const imageUrl = productList.image?.url;
-   const shouldNoIndex =
-      isFiltered ||
-      pagination.nbHits < 2 ||
-      productList.forceNoIndex;
+   const productListExemptions =
+      noIndexExemptions[encodeDeviceTitle(productList.deviceTitle ?? '')];
+   const isNoIndexExempt = itemType
+      ? productListExemptions?.itemTypes?.includes(itemType)
```

Comment:
I think this should work since double-encoding the item type won't be a problem as long as TS doesn't complain about this.
```suggestion
      ? productListExemptions?.itemTypes?.includes(encodeDeviceItemType(itemType))
```